### ISSUE TITLE
Render and publish creates issue in RSS feed XML file

### ISSUE THREAD

**Issue by fatso-jetson** [quarto-dev/quarto-actions#39]
First off, thank you for creating such an excellent publishing system. Never built my own website before, but with Quarto and gh-pages, I managed to do so in a couple of days.

I've been using the publish action to automatically update my website, but I'm having an issue with the XML file that is produced for the RSS feed. When rendering the site with Quarto itself, the XML content points to the correct URL for my site and articles/posts, e.g. "[username].github.io/...". 

However, the file that's produced by the publish action changes this web address to "github.com/[username]/[repository]/...", which results in a 404 error when links are clicked through in an RSS reader.

I am working around this issue for now by manually copying across the correct version of the XML file to my gh-pages branch, but I thought I'd flag it as an issue (a) in the hope you have some advice on how to resolve the issue, and (b) in case others are experiencing similar.

Thanks!

**Comment by cscheid** [comment]
Thanks for the report! I'll follow up with the engineer who works on the RSS part and get back to you.

**Comment by jjallaire** [comment]
We actually just made a fix in v1.1 to address this: https://github.com/quarto-dev/quarto-cli/commit/471a1b34963b77c8287f3807c3325a09ba789775

@fatso-jetson If you switch to using the v1.1 pre-release this issue should be resolved:

```yaml
- name: Set up Quarto
   uses: quarto-dev/quarto-actions/setup@v2
   with:
      version: pre-release
```

**Comment by fatso-jetson** [comment]
Works perfectly! Thank you.
